File: backend/module/Radio/src/Radio/Entity/ListenerStat.php
<?php

namespace Radio\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity
 * @ORM\Table(name="listener_stat")
 * */
class ListenerStat
{

    /**
     * @ORM\Id
     * @ORM\Column(type="integer")
     * @ORM\GeneratedValue
     * */
    protected $id;

    /**
     * @ORM\Column(type="integer")
     * */
    protected $type;

    /**
     * @ORM\Column(type="integer", nullable=true)
     * */
    protected $count;

    /**
     * @ORM\Column(type="datetime")
     * */
    protected $date;

    /**
     * @return mixed
     */
    public function getCount()
    {
        return $this->count;
    }

    /**
     * @param mixed $count
     */
    public function setCount($count)
    {
        $this->count = $count;
    }

    /**
     * @return mixed
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * @param mixed $date
     */
    public function setDate($date)
    {
        $this->date = $date;
    }

    /**
     * @return mixed
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param mixed $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return mixed
     */
    public function getType()
    {
        return $this->type;
    }

    /**
     * @param mixed $type
     */
    public function setType($type)
    {
        $this->type = $type;
    }


}

?>

File: util/model/src/main/java/hu/radio/tilos/model/ListenerStat.java
package hu.radio.tilos.model;

import javax.persistence.*;
import java.util.Date;

@Entity
@Table(name = "listener_stat")
public class ListenerStat {

    @Id
    private int id;

    @Column
    private int type;

    @Column
    private int count;

    @Column
    private Date date;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }

    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    @Override
    public String toString() {
        return "ListenerStat{" +
                "id=" + id +
                ", type=" + type +
                ", count=" + count +
                ", date=" + date +
                '}';
    }
}
